packages/dds/matrix/package.json

```diff
@@ -30,9 +30,12 @@
     "clean": "rimraf dist lib bench/dist *.tsbuildinfo *.build.log",
```

Comment:
Looks like this package, `merge-tree`, and `ink` didn't get eslint config updates. Do they still need them?

Reply:
@Josmithr 

Ah, not sure why it wasn't added. Also found out that `matrix` didn't get the eslint config updates so extended prettier for three packages. Thank you!